components/QuickstartVideo.js

```diff
@@ -0,0 +1,12 @@
+export default function QuickstartVideo() {
+    return (
+        <iframe width="410" 
+        height="220"
```

Comment:
I think you should make it responsive like larger view on larger size displays and smaller view on smaller display size.

Reply:
That's a good idea, going to add it. Thank you.

Reply:
Good idea, TailWind has that built in https://tailwindcss.com/docs/aspect-ratio